corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -267,7 +259,7 @@ def _check_for_sheet_warnings(app, sheet, headers):
 
 def _get_missing_cols(app, sheet, headers):
     expected_sheets = {h[0]: h[1] for h in headers}
-    expected_columns = _get_expected_headers(sheet, expected_sheets)
+    expected_headers = expected_sheets.get(sheet.worksheet.title, None)
     return set(expected_columns) - set(sheet.headers)
```

Comment:
Shouldn't this be
```python
    return set(expected_headers) - set(sheet.headers)
```
now?

Reply:
(fixed in later commit, 4c01ce2ed9585367a7a2228b06f3126b1fcab582)